fvm/blueprints/contracts.go

```diff
@@ -48,6 +51,28 @@ func SetContractDeploymentAuthorizersTransaction(serviceAccount flow.Address, au
 		AddArgument(arg2), nil
 }
 
+// SetContractRemovalAuthorizersTransaction returns a transaction for updating list of authorized accounts allowed to remove contracts
+func SetContractRemovalAuthorizersTransaction(serviceAccount flow.Address, authorized []flow.Address) (*flow.TransactionBody, error) {
+	arg1, err := jsoncdc.Encode(utils.AddressSliceToCadenceValue(utils.FlowAddressSliceToCadenceAddressSlice(authorized)))
```

Comment:
Maybe improve naming of `arg1` and `arg2` here, and also in `SetContractDeploymentAuthorizersTransaction`

Reply:
Good point.

Looking at this, I realize I should factor out logic and just pass in the right path. I'll push that shortly.